results.go

```diff
@@ -51,3 +51,33 @@ func PathPaymentSourceAmount(resultXDR string) (string, error) {
 
 	return StringFromStellarXdrAmount(total), nil
 }
+
+// PathPaymentIntermediatePath unpacks an envelope XDR string to
+// get the intermediate path assets.
```

Comment:
It'd be nice to have a comment about the order of the list like
```
// Returns a list of assets starting with sender's debited asset up to but not including the recipient's credited asset.
```
if that's the right order

Reply:
it's the ones in between source asset and destination asset.